src/App.tsx

```diff
@@ -1,26 +1,47 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { selector } from './app/store';
+import { getTodos } from './api';
+import { setTodos } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const [loader, setLoader] = useState(false);
+  const dispatch = useDispatch();
+  const currentTodo = selector(state => state.currentTodo);
```

Comment:
The `selector` function is used incorrectly here. It should be `useSelector` from `react-redux` to access the Redux store state. Make sure to import `useSelector` and replace `selector` with `useSelector`.